_source/_posts/2021-10-16-full-stack-java.md

```diff
@@ -0,0 +1,693 @@
+---
+layout: blog_post
+title: "Full Stack Java with React, Spring Boot, and JHipster"
+author: matt-raible
+by: advocate
+communities: [java,javascript]
+description: "This tutorial shows you how to be a full stack Java developer with React and Spring Boot."
+tags: [java, full-stack, react, spring-boot, jhipster]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: conversion
+---
+
+If you search for "Full Stack Java" on the internet, you'll likely find a lot of recruiting, courses, and jobs. Being a full stack developer can be exciting because you're learning and using a bunch of technologies. It also pays pretty well. Today, I'm going to show you how you can be a full stack Java developer with Spring Boot, React, and JHipster.
+
+If you haven't heard of JHipster, boy do I have a treat for you! JHipster started as a Yeoman application generator back in 2013 and has grown to become a development platform. It allows you to quickly generate, develop, and deploy modern web apps and microservice architectures. Today, I'll show you how to build a Flickr clone with JHipster and lock it down with OAuth and OpenID Connect (OIDC).
+
+**Prerequisites:**
+
+- [Node.js 14+](https://nodejs.org/)
+- [Java 11+](https://sdkman.io)
```

Comment:
Let's add here which distribution you suggest to use. There are a bunch, so a reader might be lost which to choose